config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'restaurants#index'
-  resources :restaurants
+  resources :restaurants, only: [
+    :index, :show, :new, :edit,
+    :create, :update, :destroy
+    ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.